### PR TITLE
Update italian translation

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
Continuation of the italian translation of Webots.

### CODE REVIEW

File: resources/translations/wb_it.ts [hunk 1]
```diff
@@ -1713,11 +1719,11 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <name>WbApplication</name>
     <message>
         <source>Could not open file: &apos;%1&apos;.</source>
-        <translation type="unfinished">Impossibile aprile il file: &apos;%1&apos;.</translation>
+        <translation>Impossibile aprile il file: &apos;%1&apos;.</translation>
     </message>
     <message>
         <source>Could not create a symbolic link of dynamic library: &apos;%1&apos;.</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossibile creare link simbolico della libreria dinamica: &apos;%1&apos;.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1726]:
```suggestion
        <translation>Impossibile creare un link simbolico della libreria dinamica: &apos;%1&apos;.</translation>
```

File: resources/translations/wb_it.ts [hunk 2]
```diff
@@ -1668,41 +1672,43 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
...
     <message>
         <source>Error when starting the HTML5 animation recording: &apos;%1&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Errore nel cominciare la registrazione dell&apos;animazione HTML5: &apos;%1&apos;</translation>
     </message>
     <message>
         <source>HTML5 animation successfully exported in &apos;%1&apos;
 </source>
-        <translation type="unfinished"></translation>
+        <translation>animazione HTML5 esportata con successo in &apos;%1&apos;
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1706]:
```suggestion
        <translation>Animazione HTML5 esportata con successo in &apos;%1&apos;
```

File: resources/translations/wb_it.ts [hunk 3]
```diff
@@ -1668,41 +1672,43 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
...
         <source>Start HTML5 animation export
 </source>
-        <translation type="unfinished"></translation>
+        <translation>Inizia esportazione animazione HTML5
+</translation>
     </message>
     <message>
         <source>HTML5 export failed</source>
-        <translation type="unfinished"></translation>
+        <translation>esportazione HTML5 fallita</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1697]:
```suggestion
        <translation>Esportazione HTML5 fallita</translation>
```

File: resources/translations/wb_it.ts [hunk 4]
```diff
@@ -1668,41 +1672,43 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     <name>WbAnimationRecorder</name>
     <message>
         <source>Streaming server already initialized.</source>
-        <translation type="unfinished"></translation>
+        <translation>Server di streaming già inizializzato.</translation>
     </message>
     <message>
         <source>Error when stopping the HTML5 animation recording: &apos;%1&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Errore nel interrompere la registrazione dell&apos;animazione HTML5: &apos;%1&apos;</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1679]:
```suggestion
        <translation>Errore nell&apos;interrompere la registrazione dell&apos;animazione HTML5: &apos;%1&apos;</translation>
```

File: resources/translations/wb_it.ts [hunk 5]
```diff
@@ -1629,31 +1633,31 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
...
     </message>
     <message>
         <source>PROTO nodes (Current Project)</source>
-        <translation type="unfinished"></translation>
+        <translation>nodi PROTO (Progetto Corrente)</translation>
     </message>
     <message>
         <source>PROTO nodes (Extra Projects)</source>
-        <translation type="unfinished"></translation>
+        <translation>nodi PROTO (Progetti Aggiuntivi)</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1660]:
```suggestion
        <translation>Nodi PROTO (Progetti Aggiuntivi)</translation>
```

File: resources/translations/wb_it.ts [hunk 6]
```diff
@@ -1629,31 +1633,31 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
...
+        <translation>Documentazione: &lt;a style=&apos;color: #5DADE2;&apos; href=&apos;%1&apos;&gt;%1&lt;/a&gt;</translation>
     </message>
     <message>
         <source>This folder lists all suitable PROTO nodes from the extra projects located in: &apos;%1&apos;.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <source>PROTO nodes (Current Project)</source>
-        <translation type="unfinished"></translation>
+        <translation>nodi PROTO (Progetto Corrente)</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1656]:
```suggestion
        <translation>Nodi PROTO (Progetto Corrente)</translation>
```

File: resources/translations/wb_it.ts [hunk 7]
```diff
@@ -1589,35 +1593,35 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
...
     </message>
     <message>
         <source>The &quot;%1&quot; icon should have a dimension of 128x128 pixels.</source>
-        <translation type="unfinished"></translation>
+        <translation>L&apos;icona &quot;%1&quot; dovrebbe avere una dimensione di 128x128 pixels.</translation>
     </message>
     <message>
         <source>This folder lists all suitable PROTO nodes from the local &apos;protos&apos; directory: &apos;%1&apos;.</source>
-        <translation type="unfinished"></translation>
+        <translation>Questa cartella elenca tutti i nodi PROTO idonei della directory locale &apos;protos&apos;: &apos;%1&apos;.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1612]:
```suggestion
        <translation>Questa sezione elenca tutti i nodi PROTO idonei della cartella locale &apos;protos&apos;: &apos;%1&apos;.</translation>
```

File: resources/translations/wb_it.ts [hunk 8]
```diff
@@ -1589,35 +1593,35 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
...
     </message>
     <message>
         <source>This folder lists all suitable PROTO nodes provided by Webots.</source>
-        <translation type="unfinished"></translation>
+        <translation>Questa cartella elenca tutti i nodi PROTO idonei forniti da Webots.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1616]:
```suggestion
        <translation>Questa sezione elenca tutti i nodi PROTO idonei forniti da Webots.</translation>
```

File: resources/translations/wb_it.ts [hunk 9]
```diff
@@ -1513,28 +1516,29 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
...
-        <translation type="unfinished">Cancella</translation>
+        <translation>Cancella</translation>
     </message>
     <message>
         <source>Add</source>
-        <translation type="unfinished">Aggiungi</translation>
+        <translation>Aggiungi</translation>
     </message>
     <message>
-        <source>Insert the identity matrix
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1527]: The source text should not be modified

File: resources/translations/wb_it.ts [hunk 10]
```diff
@@ -1326,7 +1329,7 @@ Restore initial state of the simulation. (%1+Shift+F)</source>
     </message>
     <message>
         <source>Restore &amp;Viewpoint</source>
-        <translation type="unfinished"></translation>
+        <translation>Restora &amp;Punto di Vista</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1332]:
```suggestion
        <translation>Ripristina &amp;Punto di Vista</translation>
```

File: resources/translations/wb_it.ts [hunk 11]
```diff
@@ -1032,101 +1033,103 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
...
         <source>Reset Simulation</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione</translation>
     </message>
     <message>
         <source>Reset Simulation.
 Restore initial state of the simulation. (%1+Shift+F)</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione.
+Restora lo stato iniziale della simulazione. (%1+Shift+F)</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1120]:
```suggestion
Ripristina lo stato iniziale della simulazione. (%1+Shift+F)</translation>
```

File: resources/translations/wb_it.ts [hunk 12]
```diff
@@ -1032,101 +1033,103 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
...
         <source>Reload World.
 Reload the current world file and restart the simulation. (%1+Shift+R)</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricarica Scenario.
+Ricarica lo scenario corrente e ricomincia la simulazione. (%1+Shift+R)</translation>
     </message>
     <message>
         <source>Reset Simulation</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1114]: I would rather use "azzera" than "resetta" (that is more a slang)
```suggestion
        <translation>Azzera Simulazione</translation>
```

File: resources/translations/wb_it.ts [hunk 13]
```diff
@@ -713,310 +714,310 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
...
     </message>
     <message>
         <source>Run the simulation as fast as possible. (%1+3)</source>
-        <translation type="unfinished"></translation>
+        <translation>Esegui la simulazione il più velocemente possibile. (%1+3)</translation>
     </message>
     <message>
         <source>Hide Rendering to gain better performance. (%1+4)</source>
-        <translation type="unfinished"></translation>
+        <translation>Nascondi il Rendering per ottenere migliori performance. (%1+4)</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:992]:
```suggestion
        <translation>Nascondi il Rendering per ottenere migliori prestazioni. (%1+4)</translation>
```

File: resources/translations/wb_it.ts [hunk 14]
```diff
@@ -713,310 +714,310 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
...
     </message>
     <message>
         <source>Run the simulation in real-time. (%1+2)</source>
-        <translation type="unfinished"></translation>
+        <translation>Lancia la simulazione in tempo reale. (%1+2)</translation>
     </message>
     <message>
         <source>Pause the simulation. (%1+0)</source>
-        <translation type="unfinished"></translation>
+        <translation>Pausa la simulazione. (%1+0)</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:980]:
```suggestion
        <translation>Metti in pausa la simulazione. (%1+0)</translation>
```

File: resources/translations/wb_it.ts [hunk 15]
```diff
@@ -713,310 +714,310 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
...
     </message>
     <message>
         <source>&amp;Clear Console</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Pulisci Console</translation>
     </message>
     <message>
         <source>Clears the Console.</source>
-        <translation type="unfinished"></translation>
+        <translation>Pulisce la Console.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:960]:
```suggestion
        <translation>Pulisci la Console.</translation>
```

File: resources/translations/wb_it.ts [hunk 16]
```diff
@@ -713,310 +714,310 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
     <name>WbAbstractCamera</name>
     <message>
         <source>Cannot allocate shared memory. The shared memory is required for the cameras. The shared memory of your OS is probably full. Please check your shared memory setup.</source>
-        <translation type="unfinished">Impossibile allocare la memoria condivisa necessaria per le videocamere. La memoria condivisa del vostro sistema operativo è probabilmente piena. Verificate le impostazioni della memoria condivisa.</translation>
+        <translation>Impossibile allocare la memoria condivisa necessaria per le videocamere. La memoria condivisa del vostro sistema operativo è probabilmente piena. Verificate le impostazioni della memoria condivisa.</translation>
     </message>
     <message>
         <source>Invalid &apos;fieldOfView&apos; changed to 0.7854. The field of view is limited to pi if the &apos;spherical&apos; field is FALSE.</source>
-        <translation type="unfinished"></translation>
+        <translation>Valore invalido di &apos;fieldOfView&apos; cambiato a 0.7854. Il campo visivo è limitato a pi se il campo &apos;spherical&apos;è FALSO.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:721]:
```suggestion
        <translation>Valore invalido di &apos;fieldOfView&apos; cambiato a 0.7854. Il campo visivo è limitato a pi se il campo &apos;spherical&apos; è FALSO.</translation>
```

File: resources/translations/wb_it.ts [hunk 17]
```diff
@@ -586,28 +586,29 @@ Invalid USE nodes that refer to DEF nodes defined outside the %2 node are turned
     </message>
     <message>
         <source>First item of &apos;%1&apos; field expected to be a file but is a directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il primo elemento del campo &apos;%1&apos; è supposto essere un file ma è invece una directory.</translation>
     </message>
     <message>
         <source>&apos;%1&apos; expected to be a file but is a directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>&apos;%1&apos; previsto essere un file ma è una directory.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:593]:
```suggestion
        <translation>&apos;%1&apos; previsto essere un file ma è una cartella.</translation>
```

File: resources/translations/wb_it.ts [hunk 18]
```diff
@@ -586,28 +586,29 @@ Invalid USE nodes that refer to DEF nodes defined outside the %2 node are turned
...
     </message>
     <message>
         <source>Expected VRML97 type but found &apos;%2&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Previsto un tipo VRML97 ma trovato &apos;%2&apos;</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:597]:
```suggestion
        <translation>Previsto formato VRML97 ma trovato &apos;%2&apos;</translation>
```

File: resources/translations/wb_it.ts [hunk 19]
```diff
@@ -586,28 +586,29 @@ Invalid USE nodes that refer to DEF nodes defined outside the %2 node are turned
     </message>
     <message>
         <source>First item of &apos;%1&apos; field expected to be a file but is a directory.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il primo elemento del campo &apos;%1&apos; è supposto essere un file ma è invece una directory.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:589]:
```suggestion
        <translation>Il primo elemento del campo &apos;%1&apos; deve essere un file ma è invece una cartella.</translation>
```

File: resources/translations/wb_it.ts [hunk 20]
```diff
@@ -369,75 +369,75 @@
...
     </message>
     <message>
         <source>Cannot create the Webots temporary directory &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossibile creare la directory temporanea Webots &quot;%1&quot;</translation>
     </message>
     <message>
         <source>Cannot create a directory in the Webots temporary directory &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossibile creare una directory nella directory temporanea di Webots &quot;%1&quot;</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:392]:
```suggestion
        <translation>Impossibile creare una cartella nella cartella temporanea di Webots &quot;%1&quot;</translation>
```

File: resources/translations/wb_it.ts [hunk 21]
```diff
@@ -369,75 +369,75 @@
...
     </message>
     <message>
         <source>lightest solid</source>
-        <translation type="unfinished"></translation>
+        <translation>solido più leggero</translation>
     </message>
     <message>
         <source>Cannot create the Webots temporary directory &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Impossibile creare la directory temporanea Webots &quot;%1&quot;</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:388]:
```suggestion
        <translation>Impossibile creare la cartella temporanea Webots &quot;%1&quot;</translation>
```

File: resources/translations/wb_it.ts [hunk 22]
```diff
@@ -369,75 +369,75 @@
     </message>
     <message>
         <source>types &apos;%1&apos; and &apos;%2&apos; are not matching.</source>
-        <translation type="unfinished"></translation>
+        <translation>tipi &apos;%1&apos; e &apos;%2&apos; non corrispondono.</translation>
     </message>
     <message>
         <source>%1 node cannot be inserted at the top level of the node hierarchy.</source>
-        <translation type="unfinished"></translation>
+        <translation>il nodo %1 non può essere inserito al livello superiore della gerarchia dei nodi.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:376]:
```suggestion
        <translation>Il nodo %1 non può essere inserito al livello superiore della gerarchia dei nodi.</translation>
```

File: resources/translations/wb_it.ts [hunk 23]
```diff
@@ -289,67 +289,67 @@
...
+        <translation>Carattere di fuga invalido</translation>
     </message>
     <message>
         <source>Unclosed string literal</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <source>Invalid token &quot;%1&quot;</source>
-        <translation type="unfinished"></translation>
+        <translation>Token &quot;%1&quot; invalido</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:344]:
```suggestion
        <translation>Simbolo &quot;%1&quot; invalido</translation>
```

File: resources/translations/wb_it.ts [hunk 24]
```diff
@@ -289,67 +289,67 @@
...
     </message>
     <message>
         <source>&apos;%1&apos;: error: Missing header.</source>
-        <translation type="unfinished"></translation>
+        <translation>errore &apos;%1&apos;: Intestazione mancante.</translation>
     </message>
     <message>
         <source>&apos;%1&apos;: This file was created by Webots %2 while you are using Webots %3. Forward compatibility may not work.</source>
-        <translation type="unfinished"></translation>
+        <translation>Il file &apos;%1&apos; è stato creato da Webots %2 e stai usando Webots %3. Compatibilità con versioni successive potrebbe non funzionare.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:324]:
```suggestion
        <translation>Il file &apos;%1&apos; è stato creato da Webots %2 e stai usando Webots %3. La compatibilità con versioni successive potrebbe non funzionare.</translation>
```

File: resources/translations/wb_it.ts [hunk 25]
```diff
@@ -289,67 +289,67 @@
...
     </message>
     <message>
         <source>Skipped unknown &apos;%1&apos; parameter in %2 PROTO</source>
-        <translation type="unfinished"></translation>
+        <translation>Saltato parametro sconosciuto &apos;%1&apos; nel PROTO %2</translation>
     </message>
     <message>
         <source>&apos;%1&apos;: error: Missing header.</source>
-        <translation type="unfinished"></translation>
+        <translation>errore &apos;%1&apos;: Intestazione mancante.</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:320]:
```suggestion
        <translation>&apos;%1&apos;: errore : Intestazione mancante.</translation>
```

File: resources/translations/wb_it.ts [hunk 26]
```diff
@@ -289,67 +289,67 @@
...
     </message>
     <message>
         <source>Skipped unknown &apos;%1&apos; field in %2 node</source>
-        <translation type="unfinished"></translation>
+        <translation>Saltato campo &apos;%1&apos; sconosciuto nel nodo %2</translation>
     </message>
     <message>
         <source>&apos;IS&apos; keyword is only allowed in .proto files</source>
-        <translation type="unfinished"></translation>
+        <translation>la parola chiave &apos;IS&apos; è permessa solo in files .proto</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:308]:
```suggestion
        <translation>la parola chiave &apos;IS&apos; è permessa solo in file .proto</translation>
```

File: resources/translations/wb_it.ts [hunk 27]
```diff
@@ -229,39 +229,39 @@
...
     </message>
     <message>
         <source>Expected &apos;%1&apos; but found &apos;%2&apos;</source>
-        <translation type="unfinished"></translation>
+        <translation>Previsto &apos;%1&apos; ma trovato &apos;%2&apos;</translation>
     </message>
     <message>
         <source>End of file reached while a token is expected</source>
-        <translation type="unfinished"></translation>
+        <translation>Fine del file raggiunta sebbene un token sia previsto</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:248]:
```suggestion
        <translation>Fine del file raggiunta sebbene un simbolo sia previsto</translation>
```

File: resources/translations/wb_it.ts [hunk 28]
```diff
@@ -109,75 +109,75 @@
...
     </message>
     <message>
         <source>remove item</source>
-        <translation type="unfinished"></translation>
+        <translation>rimuovi elemento</translation>
     </message>
     <message>
         <source>reset</source>
-        <translation type="unfinished"></translation>
+        <translation>resetta</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:128]:
```suggestion
        <translation>ripristina</translation>
```

File: resources/translations/wb_it.ts [hunk 29]
```diff
@@ -1032,101 +1033,103 @@ This can be caused by duplicate vertices in your mesh. Try to open your model in
...
     <message>
         <source>Reset Simulation</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione</translation>
     </message>
     <message>
         <source>Reset Simulation.
 Restore initial state of the simulation. (%1+Shift+F)</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta Simulazione.
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1119]:
```suggestion
        <translation>Azzera Simulazione.
```

File: resources/translations/wb_it.ts [hunk 30]
```diff
@@ -1891,11 +1897,11 @@ The path to this Webots project contains non 8-bit characters. Webots won&apos;t
     </message>
     <message>
         <source>Reload</source>
-        <translation type="unfinished"></translation>
+        <translation>Ricarica</translation>
     </message>
     <message>
         <source>Reset</source>
-        <translation type="unfinished"></translation>
+        <translation>Resetta</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:1904]:
```suggestion
        <translation>Azzera</translation>
```

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]
Thank you very much for your contribution!
It is never easy to translate.
I reported some minor comments and some word changes to try to use italian words instead of slang or "italianized" versions of english words.

The major issue to me seems to be the use of the `&amp;` sign (resulting in the underlined letter in the Webots menu /menu items). This is used as shortcut to open the menu from the keyboard by pressing ``<ALT+<letter>``. So even in the translation we should pay attention that the same letter is not used twice otherwise the the second one is useless and it is even better to not set it.
Could you please review all the menus in the italian version and check that these shortcuts are correctly set?

**Comment by ad-daniel** [comment]
Thank you for the feedback, I've incorporated the suggested changes.
With regards to the accelerators, that makes a lot more sense, I misunderstood their function entirely. I've fixed it so there's no doubles.

PS: the source language might require some work in that direction as well, for instance in the File menu there's 4 "O".

File: resources/translations/wb_it.ts [hunk 31]
```diff
@@ -196,75 +196,75 @@
     </message>
     <message>
         <source>Question</source>
-        <translation type="unfinished">Domanda</translation>
+        <translation>Domanda</translation>
     </message>
     <message>
         <source>edit</source>
-        <translation type="unfinished"></translation>
+        <translation>edita</translation>
```
Comment (stefaniapedrazzi) [inline resources/translations/wb_it.ts:203]:
```suggestion
        <translation>modifica</translation>
```

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]
Thank you!
There are still a couple of duplicated accelerators in "File" and "Modifica" menus, but it is fine like this.

I just have one last changes and then it's all fine for me.

**Comment by ad-daniel** [comment]
Thank you, for some reason I didn't consider the grayed out ones, should be all unique now (at least the italian ones). I also made the suggested change and marked it here as resolved but somehow it still requires it.
As for the failed test, it's the same issue that was in the master branch yesterday related to copyright dates

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you!

Is your branch up-to-date with the `webots:master` branch?

**Pull request merged**